src/tasks/onetime/DailyTask.py

```
@@ -208,6 +212,85 @@ def _open_local_path_with_default_app(self, path: str | Path):
+            "failed_details": failed_details,
+            "system_name": "OK-EF",
+            "report_type": "DAILY",
+        }
+
+    def _build_failed_details(self) -> list[dict]:
+        """从 final_summary 的 failure_details 构建失败任务明细列表。"""
+        summary = self.daily_runner.final_summary if self.daily_runner else {}
+        failure_details = summary.get("failure_details") or {}
+        per_round = summary.get("per_round") or []
+
+        # account_id -> account_user 映射（用于显示账号名）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)